swipe.js

```diff
@@ -520,7 +554,29 @@
       // Remove id from element
       clone.removeAttribute('id');
     }
-
+    function setUpPartials(containerWidth){
+      const { partial } = options
+      let partialWidthPercetage;
+      
+      /** 
+       * Setting up defaul partial Width if partial defined but erither its boolean
+       * of the partial does not seems to be in range of .01 to .20
```

Comment:
Found  several typos in above comment

Reply:
Thanks for reviewing, I have corrected the typos